### PR TITLE
Publish features/NullableReferenceTypes

### PR DESCRIPTION
**Pull request by cston**

### CODE REVIEW

**Comment by cston** [comment]
@dotnet/roslyn-infrastructure, @jaredpar please review.

File: build/config/PublishData.json [hunk 1]
```diff
@@ -21,6 +21,13 @@
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
             "channels": [ "ref-readonly" ]
         }
```
Comment (jasonmalinowski) [inline build/config/PublishData.json]: Missing comma here, which is why GitHub is flagging the syntax error.

File: build/config/PublishData.json [hunk 2]
```diff
@@ -21,6 +21,13 @@
             "vsix": [ "https://dotnet.myget.org/F/roslyn/vsix/upload" ],
```
Comment (jasonmalinowski) [inline build/config/PublishData.json]: @jaredpar So are we still uploading packages from this branch to the main feeds?? Or should we delete this entry now?

**Comment by cston** [comment]
@dotnet-bot test windows_debug_unit32_prtest please

**Pull request merged**